ail_vru_uc1_scenario/README.md

```diff
@@ -25,8 +25,12 @@ This CARMA Configuration Image creates a **AIL** (Anything-In-the-Loop) scenario
 
 ## Deployment Instructions
 ### Deployment Steps
-TODO: Complete this section
-Copy all files in the route_config directory to `/opt/carma/routes/`
+1) Copy all files in the `ail_vru_uc1_scenario/cdasimconfig/route_config/` directory to `/opt/carma/routes/`
+2) Navigate to `ail_vru_uc1_scenario` and `./build_image.sh` to build CARMA Config image. (Optional if remote image exists)
+3) Run `carma config set <carma-config-image-name>`
```

Comment:
We should add a comment mentioning that the `build_image.sh` script will print out the built image's tag, so users should use that when setting the config name.

Reply:
Seems a little like overkill but I can add it since it is not significant effort.

Reply:
Resolved